### ISSUE TITLE
Navigation arrows missing from SciGateway generated pages

### ISSUE THREAD

**Issue by Dagonite** [fiaisis/frontend#572]
### Description of the issue

The help page, privacy statement page, and the accessibility statement pages are missing navigation arrows. Will need a different implementation for these as they are rendered via SciGateway.

**Comment by Dagonite** [comment]
Had a look at this, may not be possible without a really janky implementation.
